team-config.yml

```diff
@@ -784,3 +784,12 @@ help-with-fees:
     build_notices_channel: "#pet-devops"
   tags:
     application: help-with-fees
+cui-ra:
```

Comment:
what is cui-ra? could it have a more meaningful name?

Reply:
Citizen UI Reasonable Adjustments

Reply:
why not keep cui-ra?

Reply:
CUI is product name and ra is service name, just want to differentiate this